Dockerfile

```
@@ -1,31 +1,18 @@
 # This file uses a multi-stage build strategy. The build stage sets up the nvm environment and builds configurator, while the second stage copies this into a clean container without any build tools.
 
 ## First Stage- Build
-FROM nginx as build
+FROM node:14 as build
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 
-# Pre-reqs
-COPY conf/yarn.list /tmp
-RUN apt update && \
-    apt install --no-install-recommends -y \
-        ca-certificates \
-        curl \
-        gnupg \
-        nodejs && \
-    mv /tmp/yarn.list /etc/apt/sources.list.d/yarn.list && \
-    curl https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
-    apt update -o Dir::Etc::sourcelist=/etc/apt/sources.list.d/yarn.list && \
-    apt install --no-install-recommends -y \
-        yarn && \
-    rm -rf /var/lib/apt/lists/*
-RUN curl https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash
-
 # Copy files into place
 COPY . /qmk_configurator/
+WORKDIR /qmk_configurator/
 
 # Build configurator
-RUN /bin/bash -i /qmk_configurator/bin/docker_build.sh
+RUN yarn install
+ENV VUE_APP_API_URL="/api yarn run build"
```

I know this is from the original but it looks weird. I wonder if this is one of this things that accidentally works and it should be VUE_APP_API_URL="/api" # yarn run build? because it doesn't really make sense in the way it's used to build URLs. @skullydazed was this intentional?

---

The original did not have quotes, it was a command from a shell script that would set the environment var `VUE_APP_API_URL` to `/api` and then run the command `yarn run build`. I think that should be this:

```suggestion
RUN VUE_APP_API_URL=/api yarn run build
```